### ISSUE TITLE
Liste aller Topics

### ISSUE THREAD

**Issue by RF77** [owagner/hm2mqtt#15]
Es wäre praktisch per MQTT eine Liste aller Topics zu erhalten, möglichweise noch mit ein paar Metadaten angereichert. Format z.B. JSON

Evt. ist es ja schon in der Version 0.19 erledigt:

0.19 - 2016/06/23 - owagner

    add command/list to get a channel list. Payload is a Java RegEx pattern, result is returned in "/result/list"

Nur finde ich den Release nicht und als nicht JAVA Entwickler hab ichs auf die Schnelle auch nicht geschafft, mir den neusten Code selber zu kompilieren.

**Comment by rschaten** [comment]
Ja, sieht so aus als ob das in 0.19 drin waere. Den Release haette ich auch gerne.
